nginx-log-metrics/config_data/init_database.sql

```diff
@@ -1,31 +1,31 @@
-CREATE TABLE `ngx_access_log` (
-  `client` STRING NULL,
-  `ua_platform` STRING NULL,
-  `referer` STRING NULL,
-  `method` STRING NULL,
-  `endpoint` STRING NULL,
-  `trace_id` STRING NULL FULLTEXT,
-  `protocol` STRING NULL,
-  `status` SMALLINT UNSIGNED NULL,
-  `size` DOUBLE NULL,
-  `agent` STRING NULL,
-  `access_time` TIMESTAMP(3) NOT NULL,
-  TIME INDEX (`access_time`)
+CREATE TABLE "ngx_access_log" (
```

Comment:
GreptimeDB supports both syntaxes. Why need to fix them? I ran this demo last week, looks fine.

Reply:
I'm running into error when initializing the database

ERROR:  Unexpected token while parsing SQL statement, expected: 'a table name', found: ngx_statistics: sql parser error: Expected: identifier, found: ` at Line: 1, Column: 14

Reply:
Looks like it is using the `psql`:

https://github.com/GreptimeTeam/demo-scene/blob/55e8e03c6fa0794c4f434dda4041732b91cd7d02/nginx-log-metrics/docker-compose.yml#L29